studentInfoOGB.py

```diff
@@ -0,0 +1,96 @@
+import random
+
+firstName = ["jay", "jim", "roy", "axel", "billy", "charlie", "jax", "gina", "paul",
+"ringo", "ally", "nicky", "cam", "ari", "trudie", "cal", "carl", "lady", "lauren",
+"ichabod", "arthur", "ashley", "drake", "kim", "julio", "lorraine", "floyd", "janet",
+"lydia", "charles", "pedro", "bradley"]
+
+lastName = ["blake", "holt", "meilijson", "brannon", "fieser", "line", "hellman",
+"lyndon", "wendell", "corr", "fricano", "carrigan", "giddings", "piesse", "charlwood",
+"thaler", "ambrose", "hamer", "schweig", "geiger", "kinder", "rogantin", "chevallier"
+"wang", "tiedemann", "cohn", "digel", "reheem", "dzieciololowski", "callanan",
+"lie", "oden"]
+
+def main():
+  students = [
+    Student("Larsson", 37),
+    Student("BonJovi", 55),
+  ]
+
+  printHeader()
+  selection = int(getUserSelection())
+  if selection == 0:
+    printStudentsByAge(students)
```

Comment:
```
jcharante@Thinky:~/Projects/BaxterEPCSWeek1-CR$ python -V
Python 2.7.13
jcharante@Thinky:~/Projects/BaxterEPCSWeek1-CR$ python studentInfoOGB.py 
HEADER TEXT HERE
For STUDENTS BY AGE, type 0
For STUDENTS BY LAST NAME, type 1
For STUDENTS BY FIRST NAME, type 3
Type selection and press enter:0
----Students By Age-----
s, lady, None
J, floyd, None
```

In regards to `s, lady, None`, it is clear from `line 71` that instead of `None` the age of the student was meant to be printed. 

(continued on line 35)